.github/workflows/firebase-preview.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Build
-        run: yarn && yarn build
+        run: yarn && APP_UNSAFE_ALLOW_DYNAMIC_CONFIG=1 yarn build
```

Comment:
Hm, I didn't manage to use `test--no-cleeng` config for the preview build :(

<img width="629" alt="Screenshot 2022-07-06 at 17 30 09" src="https://user-images.githubusercontent.com/27254986/177588208-0c93d6af-f27c-4351-ae28-15b98d8c1411.png">

Reply:
I think this is because the test data is only bundled with dev/test builds and the preview / github builds are production environment

Reply:
Should we maybe use dynamic configs for previews in the PR? That would help much to test different cases.